### PR TITLE
Nilot

### LINKED ISSUE #86: Fix Readme for Delete Orphaned Catalog UI Policies
The Readme file was updated incorrectly at some point. The heading and the description do not match.  Heading should be fixed to match the description of the check. See the headings just above this one for similar check description.

---
### Update set description should not be empty
Catalog UI policy should be used in either a Catalog Item or a Variable Set. Catalog UI Policies not in use should be deleted.

### PR DESCRIPTION
**Pull request by NilotpalMaitra**
Fix Readme for Delete Orphaned Catalog UI Policies

Fixes #86

### CODE REVIEW

**Comment by NilotpalMaitra** [comment]
Please do check the issue and whether it is resolving the issue or not. Thank you!

File: README.md [hunk 1]
```diff
@@ -85,8 +85,8 @@ Hard coding instance URL is not a best practice as they reduce the usability of
 ### Before Business rules should not insert() records in any tables
 Before business rules execute before the data on current record is saved to database. In case, the before business rule fails or aborts its operation, then any insert operations within the business rule becomes invalid but it cannot be reverted back as the operation would have already been completed. In order to avoid such issues, before business rules should not include insert operations.
 
-### Update set description should not be empty
-Validates the description of the update sets created is not empty as it provides the release management team better understanding what's getting pushed.
+### Catalog UI policy should be used in either a Catalog Item or a Variable Set
+Validates that Catalog UI Policies are either associated with a Catalog Item or a Variable Set. Catalog UI Policies that are not in use should be deleted as they are unnecessary.
```
Comment (SapphicFire) [inline README.md:89]: This is not the entry that needed to be updated. This should be left as is, the title that is in error is down around line 103

**Review by SapphicFire — CHANGES_REQUESTED** [review]
Thank you for your contribution. However, it seems that the file changes in this Pull Request are incorrect. Please review the files you have changed/added and ensure that all modifications are correct and valid. Closing this for now. Once you make additional changes, feel free to re-open this Pull Request or create a new one.

**Comment by NilotpalMaitra** [comment]
Thank you for clarifying the issue!, I resolved the issue in 103rd line , thank you for your patience!

**Pull request closed without merging**